vision/mnist/mlp.jl

```diff
@@ -12,9 +15,10 @@ X = hcat(float.(reshape.(imgs, :))...) |> gpu
 labels = MNIST.labels()
 # One-hot-encode the labels
 Y = onehotbatch(labels, 0:9) |> gpu
-
 m = Chain(
-  Dense(28^2, 32, relu),
+  Dense(784,32),
+  Dense(32, 32),
+  x -> Tracker.data(x),
```

Comment:
This seems like an odd unrelated change, especially because it's dropping gradients?

Reply:
I'm sorry that was a mistake. i was just playing around with it and accidentally added it. I will revert ASAP.